app/build.gradle

```diff
@@ -75,6 +75,9 @@ dependencies {
 
     implementation "io.coil-kt:coil-compose:1.3.2"
 
+    // Compose Navigation
+    implementation "androidx.navigation:navigation-compose:2.4.0-alpha10"
```

Comment:
@V9vek hey, so Im 99% certain that you don't need this dependency as `NavHost` and `NavController` are already available in this codebase without this dependency, as I tested this on a different branch. Could I ask you to try removing this dependency and see if you changes still work? Is that ok?

If it doesn't work let me know and ill approve this PR, but if it does then repush and ill approve that too as the rest of it looks fantastic! Great work!

Reply:
Hey @Kardelio, Thanks for reviewing, I removed the `line 79` dependency and checked but the build failed
It is necessary for `navArgument`, which is being used for passing arguments like `selected category` from Category Screen to Dishes Screen

Reply:
And I think you will need to pass the `selected category string` to fetch the dishes related to the clicked category because of the network request in Dishes Screen and you will also need to pass the `selected dish id` to fetch the dish details in Details screen
And for all this you will need `navArgument`